groupmne/group_model.py

```diff
@@ -25,17 +27,20 @@ def get_src_reference(subject="fsaverage", spacing="ico5", subjects_dir=None):
     src : SourceSpaces
         The source space for each hemisphere.
     """
-
+    subjects_dir = \
+        mne.utils.get_subjects_dir(subjects_dir=None, raise_error=True)
     fname_src = op.join(subjects_dir, subject, 'bem', '%s-%s-src.fif'
                         % (subject, spacing))
     if os.path.isfile(fname_src):
         src_ref = mne.read_source_spaces(fname_src)
-    elif os.path.exists(subjects_dir + subject):
+    elif os.path.exists(os.path.join(subjects_dir, subject)):
         src_ref = mne.setup_source_space(subject=subject,
                                          spacing=spacing,
                                          subjects_dir=subjects_dir,
                                          add_dist=False)
     else:
+        # XXX : I don't get the logic here. Why fetching
+        # fsaverage if subject is not fsaverage?
```

Comment:
The idea here is: find a source space on disk and read it, if not on disk but fsaverage data is available then setup the src space if not fetch it and compute it

Reply:
Ok I see now the confusion on the name of the subject ... this must be fixed